test/unit/3dtileslayerstyle.js

```diff
@@ -7,79 +7,104 @@ import PlanarView from 'Core/Prefab/PlanarView';
 import C3DTBatchTable from 'Core/3DTiles/C3DTBatchTable';
 import C3DTilesSource from 'Source/C3DTilesSource';
 import C3DTilesLayer from 'Layer/C3DTilesLayer';
+import sinon from 'sinon';
+import Fetcher from 'Provider/Fetcher';
 import Renderer from './bootstrap';
 
+import tileset from '../data/3dTiles/tileset.json';
+
+// Create a 'fake' tile content for this test purpose
+function createTileContent(tileId) {
+    const geometry = new THREE.SphereGeometry(15, 32, 16);
+    const material = new THREE.MeshBasicMaterial({ color: 0xffff00 });
+
+    // Add _BATCHID geometry attributes
+    const array = [];
+    let currentBatchId = Math.round(Math.random() * 50);
+    for (let index = 0; index < geometry.attributes.position.count; index++) {
+        array.push(currentBatchId);
+
+        // Change randomly batch id
+        if (Math.random() > 0.5) {
+            currentBatchId = Math.round(Math.random() * 50);
+        }
+    }
+    geometry.setAttribute('_BATCHID', new THREE.BufferAttribute(Int32Array.from(array), 1));
+
+    const result = new THREE.Mesh(geometry, material);
+    result.batchTable = new C3DTBatchTable();
+    result.tileId = tileId;
+
+    return result;
+}
+
 describe('3DTilesLayer Style', () => {
-    // Define crs
-    proj4.defs('EPSG:3946', '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+    let view;
+    let $3dTilesLayer;
+    let source;
+    let stubFetcherJson;
+
+    before(function () {
+        stubFetcherJson = sinon.stub(Fetcher, 'json')
+            .callsFake(() => Promise.resolve(JSON.parse(tileset)));
```

Comment:
Given the top comment, we could change by:

```suggestion
            .callsFake(() => Promise.resolve(tileset));
```

Reply:
Done. And tileset.json file removed.